docs/guide/install-cmake.rst

```diff
@@ -0,0 +1,99 @@
+.. Copyright 2003-2022 by Wilson Snyder.
+.. SPDX-License-Identifier: LGPL-3.0-only OR Artistic-2.0
+
+.. _Installation:
+
+************
+Installation
+************
+
+This section discusses how to install Verilator using cmake.
+
+.. _Tools Install:
+
+Quick Install
+=============
+
+Install Python for your platform from https://www.python.org/downloads/
+Install CMake for your platform from https://cmake.org/download/ or build it from source
+If the compiler of your choice is MSVC then install https://visualstudio.microsoft.com/downloads/
+If the compiler of your choice is Clang then install https://releases.llvm.org/download.html or build it from source
+For flex and bison use https://github.com/lexxmark/winflexbison to build and install
+For build on windows using msvc set environment variable WIN_FLEX_BISON to install directory
+For build on windows/linux/osx using ninja set environment variable FLEX_INCLUDE to directory
+containing FlexLexer.h and ensure flex/bison is available on the path
```

Comment:
Use a numbered list or otherwise, add punctuation.   Add periods to sentences and use appropriate capitalization (MSVC etc) throughout the document.